content/docs/command-reference/metrics/diff.md

```diff
@@ -56,6 +56,9 @@ lists all the current metrics without comparisons.
   $ dvc metrics diff HEAD v1 --targets t1.json t2.json
   ```
 
+  Note, that when using `--targets` it is possible to `diff` files that are not
+  specifically marked as metrics.
+
```

Comment:
We can avoid this new paragraph by changing the beginning of the --targets option desc from

`` `--targets <paths>` - limit command scope to these metric files. Using -R``
to something like
`` `--targets <paths>` - limit command scope to these metrics files (supports any file, even when not marked as metrics specifically). Using `-R` ``

Reply:
(Notice I also fixed the -R at the end, which should be in back quotes.)

Reply:
And problably we should include that info in `dvc metrics show --help`